xcat-inventory/xcclient/inventory/dbobject.py

```diff
@@ -47,6 +64,14 @@ class passwd(Base,mixin):
     __tablename__ = 'passwd'
     __table_args__ = {'autoload':True}
 
+    @classmethod
+    def getdictkey(cls):
+        return 'key.username'
+
+    @classmethod
+    def getkeys(cls):
```

Comment:
seems `mixin::getkeys` works for `passwd` class, no need to reload `getkeys ` here

Reply:
`getkeys()` in `passwd` object can keep keys order ['key','username']. The base `getkeys()` is general method.